litebird_sim/scan_map.py

```diff
@@ -90,7 +90,7 @@ def scan_map_in_observations(
     obs: Union[Observation, List[Observation]],
     maps: Dict[str, np.ndarray],
     pointings: Union[np.ndarray, List[np.ndarray], None] = None,
-    input_map_in_galactic: bool = True,
+    input_map_in_galactic: Union[bool, None] = True,
```

Comment:
What's the meaning of `None` here? Is `input_map_in_galactic` becoming a three-state variable? (`True`/`False`/`None`.) Perhaps the meaning of `None` should be documented in the docstring.